pkg/scheduler/factory.go

```diff
@@ -406,8 +334,8 @@ func (c *Configurator) CreateFromKeys(predicateKeys, priorityKeys sets.String, e
 
 	// Setup cache debugger.
 	debugger := cachedebugger.New(
-		c.nodeLister,
-		c.podLister,
+		c.informerFactory.Core().V1().Nodes().Lister(),
+		c.informerFactory.Core().V1().Pods().Lister(),
```

Comment:
This seems to be the only place were we use pod informer (other than tests). Maybe we can just create it here, instead of `cmd/kube-scheduler/app/options/options.go`

Reply:
Good catch.

It is used when registering the handlers in scheduler.go, this is what feeds the scheduler's cache.